stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/BaseMessageListHeaderViewModel.kt

```diff
@@ -53,7 +53,11 @@ public abstract class BaseMessageListHeaderViewModel constructor(
         }
     }
 
-    public fun setActiveThread(message: Message?) {
```

Comment:
Is this a breaking change, even for the old UI library, via `ChannelHeaderViewModel`?

Reply:
That's a tricky one - `activeThread` was introduced few releases ago (at that time we only had one VM in the common module) but it is used only in the new UI component module. So technically this might be a breaking change. I'll revert that and just introduce a new method

Reply:
Maybe it shouldn't be in the base class, if it's not used in the old module? All the more reason to split up & duplicate these ViewModels instead of sharing 😄

Reply:
True, but on the other hand we did multiple releases before introducing different VMs for those modules and `activeThread ` was included. I'm not sure if removing it now is safe 😢

Reply:
The base class itself should be unused by clients. We can duplicate its functionality into the inheriting classes, and then deprecate/remove the unnecessary parts.

Reply:
Agree, I think we can do it after the release because it shouldn't be super important and that won't be a breaking change